nsv13/code/game/general_quarters/dropship.dm

```diff
@@ -151,53 +151,6 @@
 		if(M && M.client && M.hud_used && length(M.client.parallax_layers))
 			M.hud_used.update_parallax(force=TRUE)
 
-//Jank ass override, because this is actually necessary... but eughhhh
```

Comment:
All of this was also getting done elsewhere? Seemed to work when I was testing stuff but obviously I can't try it with multiple occupants very well